ArgosIOS/Argos/Views/LandingPage/LandingViewModel.swift

```diff
@@ -55,6 +58,17 @@ class LandingViewModel: LoadableObject {
                         }
                     }
                     .store(in: &self.cancellables)
+                self.socketClient.$coords
+                    .sink { [weak self] coords in
+                        guard let self = self else {return}
+                        if let latitude = coords[DataTypeName.latitude.rawValue] {
```

Comment:
also your going to have to rename this to point and then maybe change the DataValue struct to take an array and then parse the latitude from the first element of the array and the longitdue from the second element